pkg/endpoints/cluster.go

```diff
@@ -29,22 +29,20 @@ import (
 
 // Properties : properties we want to be able to retrieve via REST
 type Properties struct {
-	InstallNamespace string `json:"InstallNamespace"`
-	DashboardVersion string `json:"DashboardVersion"`
-	PipelineVersion  string `json:"PipelineVersion"`
-	TriggersVersion  string `json:"TriggersVersion,omitempty"`
-	IsOpenShift      bool   `json:"IsOpenShift"`
-	ReadOnly         bool   `json:"ReadOnly"`
-	LogoutURL        string `json:"LogoutURL,omitempty"`
+	DashboardNamespace string `json:"DashboardNamespace"`
+	DashboardVersion   string `json:"DashboardVersion"`
+	PipelineNamespace  string `json:"PipelineNamespace"`
+	PipelineVersion    string `json:"PipelineVersion"`
+	TriggersNamespace  string `json:"TriggersNamespace"`
```

Comment:
can we also omitempty on this?

Reply:
Added omitempty on TriggersNamespace.
DashboardVersion, PipelineNamespace and PipelineVersion should be mandatory, shall i omitempty anyway ?

Reply:
no we can leave them as they are, just omitempty on the optional ones for now